test/helpers/kubectl.go

```diff
@@ -1339,6 +1339,23 @@ func (kub *Kubectl) generateCiliumYaml(options []string, filename string) error
 		}
 	}
 
+	if os.Getenv("KUBEPROXY") == "0" {
+		nodeIP, err := kub.GetNodeIPByLabel(K8s1)
+		if err != nil {
+			return fmt.Errorf("Cannot retrieve Node IP for k8s1: %s", err)
+		}
+
+		opts := map[string]string{
+			"global.nodePort.device":  "enp0s8",
```

Comment:
Please use this as a constant, it might not always be this device. Why `enp0s8` and not `enp0s9` for example?